server/tserver/src/test/java/org/apache/accumulo/tserver/log/LogFileKeyTest.java

```diff
@@ -98,4 +103,48 @@ public void testSortOrder() {
     }
 
   }
+
+  private void testToFromKey(int tabletId, long seq) throws IOException {
+    var logFileKey = nk(LogEvents.DEFINE_TABLET, tabletId, seq);
+    logFileKey.tablet = new KeyExtent(TableId.of("5"), new Text("b"), null);
+    Key k = logFileKey.toKey();
+
+    var converted = LogFileKey.fromKey(k);
```

Comment:
The use of `var` here obscures the types, which are not otherwise obvious. This doesn't need to be changed, but it's probably not the best use of `var`.